ci-operator/config/open-cluster-management/kube-state-metrics/open-cluster-management-kube-state-metrics-release-2.4.yaml

```diff
@@ -0,0 +1,47 @@
+base_images:
+  base:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
+  open-cluster-management_builder_go1.16-linux:
+    name: builder
+    namespace: open-cluster-management
+    tag: go1.16-linux
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: open-cluster-management
+    tag: go1.15-linux
+images:
+- dockerfile_path: Dockerfile.prow
+  from: base
+  inputs:
+    open-cluster-management_builder_go1.16-linux:
+      as:
+      - registry.ci.openshift.org/open-cluster-management/builder:go1.16-linux
+  to: kube-state-metrics
+promotion:
+  name: "2.4"
+  namespace: open-cluster-management
```

Comment:
You need to have this line in the promotion stanza in the config for the current release:

```
  disabled: true
```

Reply:
thanks @ray-harris 
For this new repo, we only maintain release branch(current `release-2.4`), no `main` or `master` branch are built against prow, so we don't need to set the `disabled: true` here.